billing_agreements.go

```diff
@@ -40,3 +40,31 @@ func (c *Client) CreateBillingAgreementToken(
 
 	return billingAgreementToken, nil
 }
+
+// CreateBillingAgreementFromToken - Use this call to create a billing agreement
+// Endpoint: POST /v1/billing-agreements/agreements
+func (c *Client) CreateBillingAgreementFromToken(
+	ctx context.Context,
+	tokenID string,
+) (*BillingAgreement, error) {
+	type createBARequest struct {
+		TokenID string `json:"token_id"`
+	}
+
+	billingAgreement := &BillingAgreement{}
+
+	req, err := c.NewRequest(
+		ctx,
+		"POST",
+		fmt.Sprintf("%s%s", c.APIBase, "/v1/billing-agreements/agreements"),
```

Comment:
i think this is just 
```suggestion
		fmt.Sprintf("%s%s", c.APIBase, "/v1/billing-agreements"),
```

Reply:
There has to be a different function for each /v1/billing-agreements/ path, so this one is strictly for the /v1/billing-agreements/agreements endpoint.